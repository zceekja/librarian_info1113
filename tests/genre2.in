add book book1.csv 111111
genre Fantasy