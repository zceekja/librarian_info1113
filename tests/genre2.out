user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: No books with genre Fantasy.

user: 